### ISSUE TITLE
[Accessibility]: Chromium source code can not be accessed?

### ISSUE THREAD

**Issue by xinali** [sourcegraph/sourcegraph#41833]
### Audit type

General/Other

### Problem description

I can access chromium source code last night, but today I can not

![image](https://user-images.githubusercontent.com/7126894/191407900-93bd3c15-ba09-42d1-ae36-c622c36d0e1c.png)

### Expected behavior

access

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.

**Comment by umpox** [comment]
Hi @xinali! This due to scheduled maintenance on Sourcegraph.com. It should be accessible now: https://sourcegraph.com/github.com/chromium/chromium
